tests/integration/components/show-header/component-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | show-header', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<ShowHeader />`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
```

Comment:
Let's make this test pass! This doesn't test much, but we at least want to make sure it can render and it shows the logo and the tagline for now. The best way to do this in a stable way, is to add `test-*` attributes to the elements, so if later we decide to change class names, our tests won't break.

So notice the comments above to add those attributes real quick, and then change your test to this.

```javascript
test('it renders', async function(assert) {
  await render(hbs`<ShowHeader />`);
  assert.dom('[data-test-element="header-logo"]).exists()
  assert.dom('[data-test-element="header-tagline"]).exists()
})
```

Load up your app and visit `http://localhost:4200/tests` and you should see that all the tests pass. Push those changes and we should be good to go!

Reply:
Still fails -- for some reason, the DONATE text is causing it to fail? Made sure that no `data-test-element` attribute was anywhere else besides the above.